frontend/src/v4/modules/viewer/betaViewer.helpers.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+export const USE_BETA_VIEWER = 'useBetaViewer';
+export const setUseBetaViewer = (useBetaViewer = false) => localStorage.setItem(USE_BETA_VIEWER, JSON.stringify(useBetaViewer));
```

Comment:
I noticed that initially the experimental feature was turned on by default. I'm guessing this is to do with the browser even though I don't think I would have set it before. Is this okay?

Reply:
by default it should be false, we want people to use the non beta viewer unless they've opted to do so